contents/docs/data/identify.mdx

```diff
@@ -0,0 +1,462 @@
+Sometimes you may want to assign multiple distinct IDs to a single user. This is helpful in scenarios where your primary distinct ID may be inaccessible. For example, if a distinct ID which is typically used on the frontend is not available in certain parts of your backend code. In this case, you can use `alias` to assign another distinct ID to the same user:
+
+In the below example, we assign the user with `distinct_id` another ID: `alias_id`. This means that any events submitted using either `distinct_id` or `alias_id` will be associated with the same user.
+
+<MultiLanguage selector="tabs">
+
+```node
+client.alias({
+    distinctId: 'distinct_id',
+    alias: 'alias_id',
+})
+```
+
+```javascript
+posthog.alias('alias_id', 'distinct_id');
+```
+
+```python
+posthog.alias('distinct_id', 'alias_id', )
+```
+
+```php
+PostHog::alias(array(
+  'distinctId' => 'distinct_id',
+  'alias' => 'alias_id'
+));
+```
+
+```ruby
+posthog.alias({
+  distinct_id: "distinct_id",
+  alias: "alias_id",
+})
+```
+
+```go
+client.Enqueue(posthog.Alias{
+  DistinctId: "distinct_id",
+  Alias: "alias_id",
+})
+```
+
+```java
+posthog.alias("distinct_id", "alias_id");
+```
+
+```bash
+curl -v -L --header "Content-Type: application/json" -d '{
+    "api_key": "<ph_project_api_key>",
+    "properties": {
+        "distinct_id": "distinct_id",
+        "alias": "alias_id"
+    },
+    "timestamp": "2020-08-16 09:03:11.913767",
+    "event": "$create_alias"
+}' https://app.posthog.com/capture/
+```
+
+</MultiLanguage>
+
+There are two requirements when assigning an `alias_id`:
+
+1. It cannot be associated with more than one `distinct_id`.
+2. The `alias_id` **must not** be the `distinct_id` of any other user. For example:
+
+<MultiLanguage selector="tabs">
+
+```node
+// ❌ Not possible. 
+client.alias({
+    distinctId: 'distinct_id',
+    alias: 'distinct_id_of_another_user',
+})
+```
+
+```javascript
+// ❌ Not possible. 
+client.alias({
+    distinctId: 'distinct_id',
+    alias: 'distinct_id_of_another_user',
+})
+```
+
+```python
+# ❌ Not possible
+posthog.alias('distinct_id_of_another_user', 'distinct_id');
+```
+
+```php
+# ❌ Not possible
+PostHog::alias(array(
+  'distinctId' => 'distinct_id',
+  'alias' => 'distinct_id_of_another_user'
+));
+```
+
+```ruby
+# ❌ Not possible
+posthog.alias({
+  distinct_id: "distinct_id",
+  alias: "distinct_id_of_another_user",
+})
+```
+
+```go
+// ❌ Not possible
+client.Enqueue(posthog.Alias{
+  DistinctId: "distinct_id",
+  Alias: "distinct_id_of_another_user",
+})
+```
+
+```java
+// ❌ Not possible
+posthog.alias("distinct_id", "distinct_id_of_another_user");
+```
+
+</MultiLanguage>
+
+> Note that when calling `alias` in the frontend SDKs, if you have set any properties onto the anonymous user, they will be merged into the user with `distinct_id`. For more details, see the FAQ on [how properties are managed when identifying anonymous users](/docs/data/identify#how-are-properties-managed-when-identifying-anonymous-users).
```

Comment:
this is new

Reply:
it's exactly the same as #### How are properties managed when merging users? why not link there?

Reply:
it's exactly the same as #### How are properties managed when merging users? why not link there?

Reply:
I think it's good to keep it as its with the rest of the context with alias.

Re having two section "#### How are properties managed when merging users?' and "####how-are-properties-managed-when-identifying-anonymous-users": While they are technically the same, it's easier to explain them as two different sections, since they apply to separate scenarios (one is frontend+anonymous users, while the other scenario is around merging in the backend)